source/deployment/bulk-loading-data-format.rst

```diff
@@ -689,3 +695,131 @@ Fields of the Post object
       <td align="center" valign="middle">Yes</td>
     </tr>
   </table>
+
+DirectChannel object
+---------------------
+
+A direct channel can have from two to eight users as members of the channel. If there are only two members, Mattermost treats it as a Direct Message channel. If there are three or more members, Mattermost treats it as a Group Message channel.
+
+Example DirectChannel object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For clarity, the object is shown using regular JSON formatting, but in the data file it cannot be spread across several lines. It must be all on one line.
+
+.. code-block:: javascript
+
+  {
+    "type": "direct_channel",
+    "direct_channel": {
+      "members": [
+        "username1",
+        "username2",
+        "username3"
+      ],
+      "header": "The Channel Header",
+    }
+  }
+
+Fields of the DirectChannel object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+      <th class="head">Validated</th>
+      <th class="head">Mandatory</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">members</td>
+      <td valign="middle">array</td>
+      <td>Must contain a list of members, with a minumum of two usernames and a maximum of eight usernames.</td>
+      <td align="center" valign="middle">Yes</td>
```

Comment:
This should be a No* where the footnote is: Not validated, but an error will be thrown in apply mode if any of the users do not exist.